css/styles.layout.css

```
@@ -5283,4 +5283,125 @@
+        z-index: 100;
+        padding: 12px 16px;
+        background: linear-gradient(135deg, rgba(77, 214, 201, 0.18) 0%, rgba(77, 214, 201, 0.1) 100%);
+        border: 1px solid rgba(77, 214, 201, 0.35);
+        border-radius: 12px;
+        backdrop-filter: blur(12px);
+        box-shadow: 0 4px 20px rgba(0, 0, 0, 0.2), 0 0 0 1px rgba(77, 214, 201, 0.1);
+        animation: hintFloatIn 0.5s cubic-bezier(0.16, 1, 0.3, 1), hintFloatBob 2.5s ease-in-out 0.5s infinite;
+        pointer-events: auto;
+      }
+
+      @keyframes hintFloatIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "hintFloatIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)